src/app/components/transaction/stacks/stacks-transaction-icon.tsx

```diff
@@ -39,15 +44,25 @@ export function StacksTransactionIcon({ transaction, ...rest }: StacksTransactio
           {...rest}
         >
           <TransactionTypeIcon
-            transaction={{ tx_type: 'contract_call', tx_status: 'pending' } as Tx}
+            transaction={
+              {
+                tx_type: 'contract_call',
+                tx_status: 'pending',
+              } as StacksTx
+            }
           />
         </DynamicColorCircle>
       );
     case PayloadType.TokenTransfer:
       return (
         <TransactionIconWrapper
           icon={StxIcon}
-          transaction={{ tx_type: 'token_transfer', tx_status: 'pending' } as Tx}
+          transaction={
+            {
+              tx_type: 'token_transfer',
+              tx_status: 'pending',
+            } as StacksTx
+          }
```

Comment:
Recognise this was here before, but if easy to fix:

Were `<TransactionIconWrapper />` was typed to only need `Pick<Tx, 'tx_status' | 'tx_type'>`, the cast wouldn't be needed.